authentication-demo-service/pom.xml

```diff
@@ -112,13 +99,12 @@
 		<dependency>
 			<groupId>org.glassfish.jersey.core</groupId>
 			<artifactId>jersey-common</artifactId>
-			<version>2.22.2</version>
 			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>io.mosip.authentication</groupId>
 			<artifactId>authentication-core</artifactId>
-			<version>1.2.0.1-B3</version>
+			<version>1.2.1-java21-SNAPSHOT</version>

```

Comment:
If it is develop branch then version should be 1.2.1-SNAPSHOT

Reply:
Done.